spec/rubocop/cop/rspec/capybara/specific_matcher_spec.rb

```diff
@@ -21,62 +21,148 @@
+       'for `have_table`' do
+      expect_offense(<<-RUBY, attr: attr)
+        expect(page).to have_css("table[#{attr}=foo]")
+                        ^^^^^^^^^^^^^^^^^{attr}^^^^^^^ Prefer `have_table` over `have_css`.
+        expect(page).to have_css("table[#{attr}]")
+                        ^^^^^^^^^^^^^^^^^{attr}^^^ Prefer `have_table` over `have_css`.
+      RUBY
+    end
+  end
+
+  %i[above below left_of right_of near count minimum maximum between
+     text id class style visible obscured exact exact_text normalize_ws
+     match wait filter_set focused disabled name placeholder options
+     enabled_options disabled_options selected with_selected
+     multiple with_options].each do |attr|
+    it 'registers an offense for abstract matcher when ' \
+       "first argument is element with replaceable attributes #{attr} " \
+       'for `have_select`' do
+      expect_offense(<<-RUBY, attr: attr)
+        expect(page).to have_css("select[#{attr}=foo]")
+                        ^^^^^^^^^^^^^^^^^^{attr}^^^^^^^ Prefer `have_select` over `have_css`.
+        expect(page).to have_css("select[#{attr}]")
+                        ^^^^^^^^^^^^^^^^^^{attr}^^^ Prefer `have_select` over `have_css`.
+      RUBY
+    end
+  end
+
+  it 'registers an offense when using abstract matcher with ' \
+     'first argument is element with multiple replaceable attributes' do
+    expect_offense(<<-RUBY)
+      expect(page).to have_css('button[disabled][name="foo"]', exact_text: 'foo')
+                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer `have_button` over `have_css`.
+      expect(page).to have_css('button:not([name="foo"][disabled])', exact_text: 'bar')
+                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer `have_button` over `have_css`.
     RUBY
   end
 
   it 'registers an offense when using abstract matcher with state' do
     expect_offense(<<-RUBY)
-    expect(page).to have_css('button[disabled]', exact_text: 'foo')
-                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer `have_button` over `have_css`.
-    expect(page).to have_css('button:not([disabled])', exact_text: 'bar')
-                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer `have_button` over `have_css`.
+      expect(page).to have_css('button[disabled]', exact_text: 'foo')
+                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer `have_button` over `have_css`.
+      expect(page).to have_css('button:not([disabled])', exact_text: 'bar')
+                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer `have_button` over `have_css`.
     RUBY
   end
 
   it 'does not register an offense for abstract matcher when ' \
     'first argument is element with nonreplaceable attributes' do
     expect_no_offenses(<<-RUBY)
+      expect(page).to have_css('button[data_disabled]')
```

Comment:
`[data-disabled]` is more realistic usage

Reply:
Indeed it is. I change to `[data-disabled]`